hibernate-core/src/main/java/org/hibernate/dialect/InformixDialect.java

```diff
@@ -312,4 +312,19 @@ public IdentityColumnSupport getIdentityColumnSupport() {
 	public String toBooleanValueString(boolean bool) {
 		return bool ? "'t'" : "'f'";
 	}
+	@Override
+	public boolean supportsUnionAll() {
+		return true;
+	}
+
+	@Override
+	public String getSelectClauseNullString(int sqlType) {
+		String typeName = getTypeName( sqlType, 1, 1, 0 );
```

Comment:
Would it be better to use `getCastTypeName()` here?

Reply:
getCastTypeName() has the same result after trim off. The original implementation is the same as in PostgreSQL81Dialect.